Chapter8_Demo/Chapter8_Demo/ViewController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+
+using UIKit;
+
+namespace Chapter8_Demo
+{
+    public partial class ViewController : UIViewController
+    {
+        protected ViewController(IntPtr handle) : base(handle)
+        {
+            // Note: this .ctor should not contain any initialization logic.
+        }
+
+        public override void ViewDidLoad()
+        {
+            base.ViewDidLoad();
+
+            //// Perform any additional setup after loading the view, typically from a nib.
+
+            test.Source = new MyTableViewDataSource();
```

Comment:
the name of the TableView should be more suggestive "RestaurantTableView" or something